metadata-ingestion/tests/integration/looker/test_looker.py

```diff
@@ -1,10 +1,12 @@
 import json
 import time
 from datetime import datetime
+from pathlib import Path
 from typing import Any, Dict, List, Optional, Sequence, Tuple, Union, cast
 from unittest import mock
 
 import pytest
+from _pytest.config import Config
```

Comment:
why `_pytest`?

Reply:
It is to silent typing lint issue for function `ingest_independent_looks`